docs/contributing.md

```diff
@@ -9,6 +9,29 @@ Thanks for contributing, you rock!
 
 When it comes to open source, there are many different kinds of contributions that can be made, all of which are valuable. Here are a few guidelines that should help you as you prepare your contribution.
 
+## Setup
```

Comment:
Since setup is important for everyone and communicates the important branching info, I moved it to the top.